youtube_dl/extractor/bostonglobe.py

```diff
@@ -0,0 +1,62 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+
+from ..utils import (
+    extract_attributes,
+    orderedSet,
+)
+
+class BostonGlobeIE(InfoExtractor):
+    _VALID_URL = r'https?(?i)://(?:www\.)?bostonglobe\.com/.*/(?P<id>[^/]+)/\w+(?:\.html)?'
+    _TEST = {
+            'url': 'http://www.bostonglobe.com/metro/2017/02/11/tree-finally-succumbs-disease-leaving-hole-neighborhood/h1b4lviqzMTIn9sVy8F3gP/story.html',
+            'info_dict': {
+                'title': 'A tree finally succumbs to disease, leaving a hole in a neighborhood - The Boston Globe',
+                'id': '5320421710001',
+                'ext': 'mp4',
+                'title': 'A tree finally succumbs to disease, leaving a hole in a neighborhood',
+                'description': 'It arrived as a sapling when the Back Bay was in its infancy, a spindly American elm tamped down into a square of dirt cut into the brick sidewalk of 1880s Marlborough Street, no higher than the first bay window of the new brownstone behind it.',
+                'timestamp': 1486877593,
+                'upload_date': '20170212',
+                'uploader_id': '245991542',
+            },
+        'md5': '0a62181079c85c2d2b618c9a738aedaf',
+        # HEAD requests produce 404 :(
```

Comment:
I guess this can be removed as the new extractor doesn't try HEAD before the actual request?